### PR TITLE
Add section about setting up one-to-one meetings.

### PR DESCRIPTION
**Pull request by xitij2000**
Adds a section for "Getting to Know Everyone" that describes the process for setting up one-to-one meetings. 

Reviewer:
[] @antoviaque

### CODE REVIEW

**Comment by xitij2000** [comment]
@antoviaque  I have updated the ``-dev`` version of the course with this content. If it seems fine I will update the main course as well.

File: html/getting_to_know_everyone_setup_meetings_1.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<p>
+    Setting up the actual meetings can be challenge when people are working
```
Comment (antoviaque) [inline html/getting_to_know_everyone_setup_meetings_1.html]: "Can be _**a**_ challenge"

File: html/getting_to_know_everyone_setup_meetings_1.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    You can also ping people on the JIRA ticket itself. This should 
+    automatically send them an email linked to the ticket where they can log 
+    time and all the essential details mentioned above can be part of the ticket
+    description. 
+</p>
+<p>
+    Whatever method you use to set up meetings, 
+    <a href="https://calendly.com/">Calendly</a> can be useful while scheduling
+    them. It lets you define your availability and share a link that lets 
+    invitees select a free time-slot on your calendar that suits them as well.
```
Comment (antoviaque) [inline html/getting_to_know_everyone_setup_meetings_1.html]: Could you put "if you want to use Calendly, you can ask <a href="mailto:xavier@opencraft.com">Xavier</a> for an account on OpenCraft's dime there."

File: html/getting_to_know_everyone_setup_meetings_1.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    from different timezones and have vastly different schedules. You can ping
+    people any way you want, preferring asynchronous means to avoid the need
+    for instant replies. 
+</p>
+<p>
+    One option is to send everyone an email inviting them to schedule a meeting.
+    The email should mention the task you've created, so they know where to log
+    the time, and remember to mention your timezone and availability in UTC. 
+    Do this as early in the sprint as possible so people have time to respond 
+    and figure out the time that suits them best.
```
Comment (antoviaque) [inline html/getting_to_know_everyone_setup_meetings_1.html]: Also mention that the task can be marked as "merged" as soon as all the meetings are scheduled, even if they haven't happened yet (to allow to spread the actual invites over multiple weeks, without having to sync it with a ticket status).

**Pull request merged**